specs/src/specs/data_square_layout.md

```diff
@@ -37,16 +36,16 @@ Specifically, blobs must begin at a new share. We note a nice property from this
 
 This, however, requires the block producer to interact with the transaction sender to provide them the starting location of their blob, so that the sender can sign over the commitment based on that starting location. This can be done selectively, but is not ideal as a default for e.g. end-user wallets.
 
-### Non-Interactive Default Rules
+### Non-Interaction Rules
 
-As a non-consensus-critical default, we can impose one additional rule on blob placement to make the possible starting locations of blobs sufficiently predictable and constrained such that users can deterministically compute subtree roots without interaction:
+We impose one additional rule on blob placement to make the possible starting locations of blobs sufficiently predictable and constrained such that users can deterministically compute subtree roots without the need to interact with the block proposer:
 
-> Blobs start at an index that is a multiple of the blob minimum square size. The blob minimum square size is the smallest square that can contain the blob in isolation (i.e. a square with only this blob and no other transactions or blobs).
+> Blobs must start at an index that is a multiple of the `SubtreeWidth`. The `SubtreeWidth` is the length of the blob in shares, divided by the [`SubtreeRootThreshold`](https://github.com/celestiaorg/celestia-app/blob/v1.0.0-rc2/pkg/appconsts/v1/app_consts.go#L6) ([implementation here](https://github.com/celestiaorg/celestia-app/blob/v1.0.0-rc2/pkg/shares/non_interactive_defaults.go#L94-L116)).
```

Comment:
Isn't `SubtreeWidth` the length of the blob divided by the `SubtreeRootThreshold` and then to the power of two. If we don't want the power of two part then really it's the `SubtreeDepth` (or height lol depending on who you ask).

Also as an additional constraint, a blob may start at the start of the next row which may not necessarily be a multiple of the SubtreeWidth or if SubTreeWidth is too large we use the `BlobMinSquareSize()`

Reply:
good point, and yeah I agree. I rushed this through last night to at least get an up to date version since other teams are already building on it and need that, but was thinking the same thing. Visuals would go a long way here. https://github.com/celestiaorg/celestia-app/issues/1913

5fed294d871f85f0839540859c4cec56acc3dd1a